src/core/errors/contract_address_errors.rs

```diff
@@ -31,4 +31,6 @@ pub enum ContractAddressError {
     CairoRun(#[from] CairoRunError),
     #[error(transparent)]
     VirtualMachine(#[from] VirtualMachineError),
+    #[error("Could not remove suffix from builtin")]
```

Comment:
Is this Error really needed? can we re-use an existing one?

Reply:
I didn't find useful any existing error. Let me know if you think I'm wrong.